reference/src/clientimpl/src/sa_process_common_encryption.c

```diff
@@ -114,28 +102,24 @@ sa_status sa_process_common_encryption(
                     break;
                 }
 
+                if (samples[i].out->context.clear.offset > samples[i].out->context.clear.length) {
```

Comment:
Are these some of the integer overflow tests from the other PR?

Reply:
Yes. Removed